pkg/cloud/azure/services/groups/groups.go

```diff
@@ -42,7 +42,7 @@ func (s *Service) CreateOrUpdate(ctx context.Context, spec azure.Spec) error {
 // Delete deletes the resource group with the provided name.
 func (s *Service) Delete(ctx context.Context, spec azure.Spec) error {
 	klog.V(2).Infof("deleting resource group %s", s.Scope.MachineConfig.ResourceGroup)
-	future, err := s.Client.Delete(ctx, s.Scope.MachineConfig.ResourceGroup)
+	future, err := s.Client.Delete(ctx, s.Scope.MachineConfig.ResourceGroup, "Microsoft.Compute/virtualMachines")
```

Comment:
Is there a const in the azure API we can use for this?

Reply:
I found one for `Microsoft.Compute` but not for the `/virtualMachines` unfortunately. This parameter is the force delete parameter:
```
forceDeletionResourceTypes - the resource types you want to force delete. Currently, only the following is supported: forceDeletionResourceTypes=Microsoft.Compute/virtualMachines,Microsoft.Compute/virtualMachineScaleSets
```

Reply:
Ok, can we make our own const for this then at the top and add an appropriate comment with the context from that message